Source/items.cpp

```diff
@@ -2749,6 +2749,8 @@ void CalcPlrGraphics(Player &player, PlayerWeaponGraphic animWeaponId, PlayerArm
 		int8_t numberOfFrames;
 		int8_t ticksPerFrame;
 		player.getAnimationFramesAndTicksPerFrame(graphic, numberOfFrames, ticksPerFrame);
+		if (graphic == player_graphic::Death && (player._pgfxnum & 0xF) != 0)
+			assert("CalcPlrGraphics: Death graphic requested while weapon bits are still set");

```

Comment:
Best to take out the assert. We don't want the game crashing here if it can be reached via remote player actions.

```suggestion
```